docs/csharp/csharp-7.md

```diff
@@ -16,64 +16,539 @@ ms.assetid: fd41596d-d0c2-4816-b94d-c4d00a5d0243
 # What's new in C# 7
 
 C# 7 adds a number of new features to the C# language:
-
 * [`out` variables](#out-variables)
```

Comment:
I checked with @mairaw and while it does look a bit strange, it's the right styling. Otherwise, the keyword might get localized.